encrypted-media/drm-mp4-playback-temporary.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<meta name="timeout" content="long">
+<html>
+  <head>
+    <meta charset=utf-8>
+    <title>Encrypted Media Extensions ClearKey Success</title>
+    <link rel="help" href="https://w3c.github.io/encrypted-media/">
+
+    <script src=/resources/testharness.js></script>
```

Comment:
They could, though we have then to be sure they are loaded before the rest of the test runs. I don't see a problem loading them all in the HTML. (This HTML is probably going to become a small number of template files with a Makefile to build all the tests).